### PR TITLE
added logo, updated  index.js and style.css in main

### PR DESCRIPTION
**Pull request by ansarraja**

### CODE REVIEW

File: src/components/Header/style.css [hunk 1]
```diff
@@ -1,2 +1,9 @@
 .App-header {
+ display: flex;
```
Comment (JonHarrison) [inline src/components/Header/style.css]: Please undo these changes

File: src/components/Main/index.js [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 
 import './style.css'
 
 const Main = () => {
+
   return (
-    <main className="App-main" id="main">Main</main>
-  )
-}
+    <div className="container">
```
Comment (JonHarrison) [inline src/components/Main/index.js]: Please keep this in <main> and use App-main for the surrounding css class

File: src/components/Main/style.css [hunk 3]
```diff
@@ -1,2 +1,9 @@
-.App-main {
+.App-header {
```
Comment (JonHarrison) [inline src/components/Main/style.css]: Please keep this as .App-main

**Review by lokiheim — APPROVED** [review]

**Review by JonHarrison — CHANGES_REQUESTED** [review]
please address the requested changes

File: src/components/Header/style.css (hunk 1 above)
Comment (ansarraja) [reply src/components/Header/style.css]: done

File: src/components/Main/index.js (hunk 2 above)
Comment (ansarraja) [reply src/components/Main/index.js]: done

File: src/components/Main/style.css (hunk 3 above)
Comment (ansarraja) [reply src/components/Main/style.css]: done

**Review by JonHarrison — APPROVED** [review]
